tests/lib/rules/no-hooks-from-ancestor-modules.js

```diff
@@ -48,6 +48,12 @@ ruleTester.run("no-hooks-from-ancestor-modules", rule, {
         QUnit.module("module", function(hooks) { hooks.afterEach(function() {}); });
         `,
         `
+        QUnit.module("module", {}, function(hooks) { hooks.beforeEach(function() {}); });
+        `,
+        `
+        QUnit.module("module", makeOptions(), function(hooks) { hooks.beforeEach(function() {}); });
+        `,
+        `
```

Comment:
Shall we add in invalid test case to ensure the rule still works when this options parameter is present?

Reply:
Good idea. Done!